src/ProvidedTypes.fsi

```diff
@@ -368,6 +389,7 @@ namespace ProviderImplementation.ProvidedTypes
         /// Get or set a utility function to log the creation of root Provided Type. Used to debug caching/invalidation.
         static member Logger: (string -> unit) option ref
 
+        member PatchDeclaringType : x:ProvidedTypeDefinition -> unit
```

Comment:
Please add a comment here

Reply:
You mean an xml doc comment, or why I need it?

Reply:
added xmldoc